cypress/e2e/BiographyTestsFrontEnd/index.js

```diff
@@ -0,0 +1,62 @@
+/// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Then('page title is {string}',(title)=>{
+cy.get(`h1:contains("${title}")`).should('be.visible').and('include.text',title)
+})
+
+And('the individuals title is {string}',(chiefCommunicationTeam)=>{
+cy.get(`div:contains("${chiefCommunicationTeam}")`).should('be.visible').and('include.text',chiefCommunicationTeam)
```

Comment:
the locator is too generic - if any other div tag on a page has the same string, then this test is not reliable 
the correct locator for this would be div.profile-title ( or div[class="profile-title"], which is the same thing in css)